README.md

```diff
@@ -39,19 +39,29 @@ You can also split a command into several ones, or separate the environment vari
 {
   "scripts": {
     "parentScript": "cross-env GREET='Joe' npm run childScript",
-    "childScript": "echo Hello $GEET"
+    "childScript": "echo Hello $GREET"
     }
 }
 ```
 
-Where `childScript` holds the actual command to execute and `parentScript` sets the environment variables to use. 
+Where `childScript` holds the actual command to execute and `parentScript` sets the environment variables to use.
 Then instead of run the childScript you run the parent. This is quite useful for launching the same command with different env variables or when the environment variables are too long to have everything in one line.
 
+If you need to translate environment variables before setting them, use [`cross-var`])(https://github.com/elijahmanor/cross-var).  Example:
```

Comment:
This is formatted incorrectly:

<img width="669" alt="screen shot 2017-02-15 at 4 17 18 pm" src="https://cloud.githubusercontent.com/assets/1500684/22999736/421d2a5a-f39a-11e6-85f6-eb71beed47cb.png">

Try:

```
If you need to translate environment variables before setting them, use [`cross-var`](https://github.com/elijahmanor/cross-var))
```